src/views/forms/FormGeneralSettings/FormGeneralSettings.vue

```diff
@@ -42,6 +40,15 @@
                         </template>
                     </FormRow>
 
+                    <FormRow>
+                        <template v-slot:label> {{ $t('set_explorer_link') }}: </template>
```

Comment:
Maybe for consistency with the labels above, we should change it to this:
```suggestion
                        <template v-slot:label> {{ $t('form_label_explorer_link') }}: </template>
```
Then the translation key should be also updated

Reply:
Updated.